Puzzlepart/prosjektportalen365
# Converting SP dates to readable excel date format

### Your checklist for this pull request


### Description

When exporting project information with dates from "Porteføljeoversikt", the dates were not recognized by excel due to the format. 
The solution: Convert the Sharepoint date format to a standard JS date object recognized by excel. 

### How to test

1. Navigate to "Porteføljeoversikt", add start and end date columns, then export to excel. 
2. The dates should now be recognized by excel. 


### Relevant issues (if applicable)
fixes #487 

💔Thank you!
